src/views/activity/activity.view.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from "react";
+
+import TrasnsactionCard from "src/views/activity/components/transaction-card/transaction-card.view";
+import useActivityStyles from "src/views/activity/activity.styles";
+import Typography from "src/views/shared/typography/typography.view";
+import Header from "src/views/shared/header/header.view";
+import { demoData } from "src/views/activity/demo-data";
+
+const Activity = () => {
+  const classes = useActivityStyles();
+  const [listAll, setListAll] = useState(true);
+  const pendingActivities = demoData.filter(
+    (data) => data.status !== "complete" && data.status !== "error"
+  );
+  const transactionsList = listAll ? demoData : pendingActivities;
+
+  const changeList = () => {
+    setListAll(!listAll);
+  };
+
+  return (
+    <>
+      <Header title={"Activity"} />
+      <div className={classes.selectorBoxes}>
+        <div
+          className={`${classes.selectorBox} ${!listAll ? classes.inactiveBox : ""}`}
+          onClick={!listAll ? changeList : undefined}
```

Comment:
It's not a good practice to add or remove handlers conditionally:

```
const onDisplayAll = () => setListAll(true)
const onDisplayPending = () => setListAll(false)

onClick={onDisplayAll}
onClick={onDisplayPending}
```

Doing this we avoid having conditional handlers and the component state won't change if you click more than once a button, so it's not going to trigger a new render.